apps/testingaccessibility/src/utils/algolia/highlight.tsx

```diff
@@ -0,0 +1,25 @@
+import type {parseAlgoliaHitHighlight} from '@algolia/autocomplete-preset-algolia'
```

Comment:
this feels like a component rather than a util? it returns a component anyway. 

would love to see some tests crop up :)

Reply:
moved to components in https://github.com/skillrecordings/products/pull/226/commits/20b004785d4b542257766d5501faf7a2e2ce2a6e 👍 tests coming up, I still have a lot to learn in regards to testing :)